buildshiprun/handler.go

```diff
@@ -143,6 +152,42 @@ func deployFunction(deploy deployment, gatewayURL string, c *http.Client) (strin
 	return string(buildStatus), err
 }
 
+func reportStatus(status string, desc string, statusContext string) {
+	owner := os.Getenv("Http_Owner")
+	repo := os.Getenv("Http_Repo")
+	commitId := os.Getenv("Http_Commit")
+	url := os.Getenv("Http_Url")
+
+	installlationId, _ := strconv.Atoi(os.Getenv("Http_Installationid"))
```

Comment:
To avoid confusion with casing can we use: `Http_Installation_Id`?

Reply:
`Installation_Id` becomes `Http_Installation_id`, if It is not an intended behaviour I can make a pull request in the `faas`